client/src/views/invitations/InvitationsPage.vue

```diff
@@ -0,0 +1,923 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-page class="invitations-page">
+    <div
+      v-if="isSmallDisplay"
+      class="switch-view"
+      @click="openInvitationSwitchModal"
+    >
+      <ion-button
+        v-if="view === InvitationUserView.EmailInvitation"
+        class="switch-view-button email-button"
+      >
+        <ion-icon
+          :icon="mailUnread"
+          class="switch-view-button__icon"
+        />
+        {{ $msTranslate('InvitationsPage.emailInvitation.tab') }}
+        <span class="switch-view-button__count">{{ invitations.length }}</span>
+        <ion-icon
+          :icon="chevronDown"
+          class="switch-view-button__chevron"
+        />
+      </ion-button>
+
+      <ion-button
+        v-else
+        class="switch-view-button pki-button"
+      >
+        <ion-icon
+          :icon="idCard"
+          class="switch-view-button__icon"
+        />
+        {{ $msTranslate('InvitationsPage.pkiRequests.tab') }}
+        <span class="switch-view-button__count button-small">{{ pkiRequests.length }}</span>
+        <ion-icon
+          :icon="chevronDown"
+          class="switch-view-button__chevron"
+        />
+      </ion-button>
+    </div>
+
+    <ion-content class="content-scroll">
+      <div class="toggle-view-container">
+        <div
```

Comment:
We could consider an intermediate window width state in which the button text could be hidden. Right now under 930px wide the display is cropped
<img width="541" height="58" alt="Screenshot 2025-09-19 at 11 17 39" src="https://github.com/user-attachments/assets/613c9815-8829-4db3-9fef-09356f317719" />